contracts/test/veLON/Convert.t.sol

```diff
@@ -3,23 +3,29 @@ pragma solidity 0.7.6;
 
 import "@openzeppelin/contracts/math/SafeMath.sol";
 import "contracts/test/veLON/Setup.t.sol";
-import "contracts/test/mocks/MockxxxLon.sol";
+import "contracts/test/mocks/MockMigrateStake.sol";
 
 contract TestVeLONDeposit is TestVeLON {
     using BalanceSnapshot for BalanceSnapshot.Snapshot;
     using SafeMath for uint256;
 
     function testEnableAndDisableConversion() public {
-        MockxxxLon xxxLon = new MockxxxLon(address(lon));
+        MockMigrateStake migrateStake = new MockMigrateStake(address(lon));
+        _stakeAndValidate(user, DEFAULT_STAKE_AMOUNT, DEFAULT_LOCK_TIME);
```

Comment:
Minor: since `enabled` is checked at the very beginning of `convert()`, no need to stake in this test.

Reply:
Repositioned in [0ffca15](https://github.com/consenlabs/tokenlon-contracts/pull/70/commits/0ffca157bf0cf7cea5483e7f9b2e45b332e3aab2)